person_capture/gui_app.py

```
@@ -6012,27 +6026,27 @@ def save_hit(
-                            else self._even_hdr_crop_xyxy(source_crop_xyxy, source_size)
-                        )
+                        # HDR archive crops are encoded from 4:2:0/10-bit source video.
+                        # Keep x/y/w/h even for AVIF and MKV; odd crop origins can shift
+                        # chroma and show up as blue/purple blotches in dark regions.
+                        hdr_crop_xyxy = self._even_hdr_crop_xyxy(source_crop_xyxy, source_size)
                         if archive_q is not None:
+                            archive_item = {
+                                "type": "hdr_archive",
+                                "path": hdr_out_path,
+                                "frame_idx": int(idx),
+                                "frame_pts_sec": frame_pts_sec,
```

Ruff (0.15.11)

[warning] Function definition does not bind loop variable `frame_pts_sec`

(B023)